arches_her/pkg/preliminary_sql/select_sources.sql

```diff
@@ -1,7 +1,9 @@
 INSERT INTO map_sources(name, source)
```

Comment:
I think this is okay for now, but it will need to be moved to a migration as HER is transitioned from a project to an Arches Application.

Reply:
@csmith-he Ah.. just saw your comment above. You're already on it.

Reply:
@chiatt Definitely the plan for it to be moved to a migration for the Arches Application

Reply:
sql file is now finalised and will be upgraded to migration separately